wled00/data/pixart/pixart.js

```
@@ -227,11 +227,11 @@ function isValidBase64Gif(string) {
   const base64jpgPattern = /^data:image\/jpg;base64,([A-Za-z0-9+/:]{4})*([A-Za-z0-9+/:]{3}=|[A-Za-z0-9+/:]{2}==)?$/;
   const base64webpPattern = /^data:image\/webp;base64,([A-Za-z0-9+/:]{4})*([A-Za-z0-9+/:]{3}=|[A-Za-z0-9+/:]{2}==)?$/;
   */
-  //REMOVED, Any image appear to work as long as it can be drawn to the canvas. Leaving code in for future use, possibly
-  if (1==1 || base64gifPattern.test(string) || base64pngPattern.test(string) || base64jpgPattern.test(string) || base64webpPattern.test(string)) {
+  // REMOVED, Any image appear to work as long as it can be drawn to the canvas. Leaving code in for future use, possibly
+  if (1 == 1 || base64gifPattern.test(string) || base64pngPattern.test(string) || base64jpgPattern.test(string) || base64webpPattern.test(string)) {
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)